tests/tasks/Cloudwatch/cw-metric.yaml

```diff
@@ -4,18 +4,19 @@ metadata:
   name:  cloudwatch
   namespace: tekton-pipelines
 spec:
-  description: "cw task to push metrics relying on generate task."
+  description: "cw task to push one data point to AWS CloudWatch."
   params:
     - name: region
       default: "us-west-2"
     - name: desired-nodes
     - name: load-result
   steps:
   - name: cw-emit
-    image: alpine/k8s:1.22.6
+    image: amazon/aws-cli
     script: |
+      aws sts get-caller-identity
       if [[ "$(params.load-result)" -eq 0 ]]; then
-      aws cloudwatch --region $(params.region) put-metric-data --metric-name $(params.desired-nodes)_nodes --namespace Cl2Load-Tests --unit Count --value 1 --dimensions TypeOfTest=Cl2_Test
+      aws cloudwatch --region $(params.region) put-metric-data --metric-name cl2-loadtest --namespace k8s-version --unit Count --value 1 --dimensions Nodes=$(params.desired-nodes)
       else
-      aws cloudwatch --region $(params.region) put-metric-data --metric-name $(params.desired-nodes)_nodes --namespace Cl2Load-Tests --unit Count --value 0 --dimensions TypeOfTest=Cl2_Test 
+      aws cloudwatch --region $(params.region) put-metric-data --metric-name cl2-loadtest --namespace k8s-version --unit Count --value 0 --dimensions Nodes=$(params.desired-nodes)
```

Comment:
Per @hakuna-matatah's comment, we still want to keep dimensions `TypeOfTest`. We can use `TypeOfTest=load` and add `TypeOfTest=density` in the future.

Reply:
I don't think `type of test` dimension is required as we are explicitly having different metric name for different cl2 test type ( I suggested in my earlier comment like for example -  metric-name to be `cl2-loadtest`) . Makes sense ?

Reply:
So `type of test` is in the metrics name instead of using a dimension. We will metrics like `cl2-loadtest` and `cl2-densitytest` (in the future).
It makes sense. It seems I misinterpreted your earlier comment.